sources/platform/actors/development/permissions/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Permissions
+description: Learn how to declare and manage permissions for your Actor, what access levels mean, and how to build secure, trusted Actors for Apify users.
+sidebar_position: 7.5
+slug: /actors/development/permissions
+---
+
+**Learn how to declare and manage permissions for your Actor, what access levels mean, and how to build secure, trusted Actors for Apify users.**
+
+---
+
+Every time a user runs your Actor, it runs under their Apify account. **Actor Permissions** is an Actor level setting that defines the level of access your Actor needs to be able to run. This gives users transparency and control over what data your Actor can access, building trust in your tools.
```

Comment:
Re _your Actor_: Keep in mind that this documentation should serve _all_ users, not just publishers. You might not own any Actors and still be interested in how you are protected.

I would consider **three main parts** of the documentation:

- **What are Actor permissions.** This should be for all users. Somewhat corresponding to Nish's blog post.
- **Actor development and permissions.** This should be for Actor creators / publishers. Somewhat corresponding to the community documentation (excluding ofc stuff from the first part).
- **Migration guide for existing full permission Actors.** Targeted at existing Actor publishers. Most of this will be already covered in the previous section but there are a few migration-specific parts (such as how to take care of existing shared cache).